docs/Caching/Redis.md

```diff
@@ -0,0 +1,33 @@
+# Redis Caching
+
+Redis is a key-value style data-structures store. The key length as well as the value can be as large as 512 MB. The 'value' in Redis can be one of the multiple [data structures](https://redis.io/topics/data-types). 
+ 
+ Performance of Redis depends on whether it is running in persistent mode (saving data to disk) or in-memory mode (all contents are within the specified memory limits). Default configurations run in persistent mode. However, depending on use-case (data structures stored, amount of data stored etc.), performance of Redis can be between "a little slower" to "just a little faster" than memcached. However, implementation of Redis is way more powerful because of the data-structures implementation.
+ 
+ ## Usage of Predis library
+ [Predis](https://github.com/nrk/predis) is the most powerful and popular client library for Redis written in PHP. QCubed uses Predis for using Redis as cache. QCubed's cache system cannot use Redis unless you install Predis. It is also a suggested package in our `composer.json`.
```

Comment:
Which statement do you think is my personal opinion? The one about "most powerful library" (I guess)?